src/Sulu/Bundle/WebsiteBundle/Controller/DefaultController.php

```diff
@@ -64,4 +62,38 @@ public function redirectAction(Request $request)
     {
         return new RedirectResponse($request->get('url'), 301);
     }
+
+    /**
+     * Resolve the redirect URL, appending any additional path data
+     *
+     * @param string $url Original webspace URI
+     * @param string $redirectUrl  Redirect webspace URI
+     * @param string $requestUri The actual incoming request URI
+     *
+     * @return string URL to redirect to
+     */
+    protected function resolveRedirectUrl($portalUrl, $redirectUrl, $requestUri)
```

Comment:
maybe we should delete the redirectWebspaceAction and move resolveRedirectUrl into our ContentRouteProvider.php and call then the normal redirectAction at https://github.com/sulu-cmf/sulu/blob/b40ddca685e0e53a5d95b00f6dcf1826c41c8ac7/src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php#73#L73